connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/PlanGenerationTestSuite.scala

```diff
@@ -516,6 +516,10 @@ class PlanGenerationTestSuite
     simple.where("a + id < 1000")
   }
 
+  test("between expr") {
+    simple.selectExpr("rand(123) BETWEEN 0.1 AND 0.2")
```

Comment:
can we add this test to a golden file test as well? This one is only for Spark Connect, we need to test the classic Spark SQL too.

Reply:
I mean this golden file: sql/core/src/test/resources/sql-tests/inputs/predicate-functions.sql

Reply:
ping @dbatomic , this is not addressed yet.